Venmo/src/main/java/com/braintreepayments/api/VenmoPaymentMethodUsage.java

```diff
@@ -0,0 +1,21 @@
+package com.braintreepayments.api;
+
+import androidx.annotation.IntDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * Enum representing different types of supported Braintree payment methods.
```

Comment:
This makes it sound like this enum will contain payment methods like PayPal, Venmo, Card, etc. On iOS, the equivalent doc string says "Usage type for the tokenized Venmo account". Maybe we could do the same here?